tests/package.json

```diff
@@ -14,7 +14,7 @@
   "license": "ISC",
   "dependencies": {
     "@holo-host/cryptolib": "^0.3.2",
-    "@holochain/tryorama": "0.6.2",
+    "@holochain/tryorama": "0.7.0",
```

Comment:
FYI be careful there was a bug in 0.7.0 where `network_seed` changes were not integrated: https://github.com/holochain/tryorama/issues/145
that's just been resolved and released in 0.8.0. 
It can cause tests to fail